include/nsysccr/cdc.h

```diff
@@ -120,6 +121,12 @@ typedef enum CCRCDCExt
    CCR_CDC_EXT_UNK4        = 4,
 } CCRCDCExt;
 
+typedef enum CCRCDCDrhStateEnum
+{
+    CCR_CDC_SYS_DRH_STATE_ECO  = 0x04,
+    CCR_CDC_SYS_DRH_STATE_CAFE = 0xFF,
```

Comment:
```suggestion
   CCR_CDC_SYS_DRH_STATE_UNK0   = 0x00,
   CCR_CDC_SYS_DRH_STATE_UNK1   = 0x01,
   CCR_CDC_SYS_DRH_STATE_UNK2   = 0x02,
   CCR_CDC_SYS_DRH_STATE_UNK3   = 0x03,
   CCR_CDC_SYS_DRH_STATE_ECO    = 0x04,
   CCR_CDC_SYS_DRH_STATE_UNK7F  = 0x7F,
   CCR_CDC_SYS_DRH_STATE_CAFE   = 0xFF,
```
These are all the valid states the DRH firmware supports.